src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -58,7 +58,7 @@
             </div>
         }
 
-        <form asp-resource="@Model.Review" asp-requirement="@ReviewOwnerRequirement.Instance" class="form-inline float-right ml-2" method="post" asp-page-handler="ToggleClosed">

```

Comment:
Don't we check this in the manager?

Reply:
I have pushed an update to take care of this.